.storybook/style/github_style.css

```diff
@@ -0,0 +1,961 @@
+}
+
+.md .octicon {
+  display: inline-block;
+  fill: currentColor;
+  vertical-align: text-bottom;
+}
+
+.md .anchor {
+  float: left;
+  line-height: 1;
+  margin-left: -20px;
+  padding-right: 4px;
+}
+
+.md .anchor:focus {
+  outline: none;
+}
+
+.md h1 .octicon-link,
+.md h2 .octicon-link,
+.md h3 .octicon-link,
+.md h4 .octicon-link,
+.md h5 .octicon-link,
+.md h6 .octicon-link {
+  color: #1b1f23;
+  vertical-align: middle;
+  visibility: hidden;
+}
+
+.md h1:hover .anchor,
+.md h2:hover .anchor,
+.md h3:hover .anchor,
+.md h4:hover .anchor,
+.md h5:hover .anchor,
+.md h6:hover .anchor {
+  text-decoration: none;
+}
+
+.md h1:hover .anchor .octicon-link,
+.md h2:hover .anchor .octicon-link,
+.md h3:hover .anchor .octicon-link,
+.md h4:hover .anchor .octicon-link,
+.md h5:hover .anchor .octicon-link,
+.md h6:hover .anchor .octicon-link {
+  visibility: visible;
+}
+
+.md {
+  -
```

Comment:
this accidental `-` is breaking the build

Reply:
fixed up.